app/controllers/expenses_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+class ExpensesController < ApplicationController
+  # respond_to :json
+
+  def index
+    @budget_id = params[:budget_plan_id]
```

Comment:
```suggestion:
    budget_id = params[:budget_plan_id]
```

nit: this doesn't have to be a class variable since it's only used in this method.

Reply:
agreed!